standard-notes-to-enex.py

```diff
@@ -21,14 +23,62 @@ def text_from_html(title, text):
 <en-export export-date="20171228T194211Z" application="Evernote" version="Evernote Mac 6.13.3 (455969)">
 '''
 
+tag_notes_links = {}
+# first we build the tag hash
 for key, item in enumerate(data['items']):
-    if 'title' in item['content'] and 'text' in item['content']:
+    if item['content_type'] == 'Tag':
+        for inner_key, reference in enumerate(item['content']['references']):
+            if 'uuid' not in reference:
+                print(f"Missing UUID for one entry in {reference} ... this can happen sometimes ... reason so far unclear")
+                continue
+            if reference['uuid'] in tag_notes_links:
+                tag_notes_links[reference['uuid']].append(html.escape(item['content']['title']))
+            else:
+                tag_notes_links[reference['uuid']] = [html.escape(item['content']['title'])]
+
+
+#print (json.dumps(tag_notes_links, indent=2))
+#quit()
+
+counter = 0
+for key, item in enumerate(data['items']):
+ if item['content_type'] == 'Note':
```

Comment:
There's bad indentation.

Reply:
unclear what you mean by bad indentation ....? Maybe your editor got it wrong? Looks fine to me ...?

Anyhow, I closed the request and send you an new one.

Please have a look at it.

Also you can accept the pull-request and modify it afterwards if you want to have the code differently in your repo. Just wanted you to get the credit and keep it centralised

Reply:
sry, my bad. Saw it was just one space instead of usually 4. fixed